cishouseholds/derive.py

```diff
@@ -2054,7 +2238,10 @@ def flag_records_for_student_v0_rules() -> F.Column:
         (F.col("age_at_visit") >= F.lit(17))
         & (
             F.col("work_status_v0").isNull()
-            | F.col("work_status_v0").isin("Employed", "Furloughed (temporarily not working)")
+            | F.col("work_status_v0").isin(
```

Comment:
not sure if this is correct

conlfuence says:
self_employed_rules_v1_a: if work_status_v1== 1